config/testdata/good_env_config.yaml

```diff
@@ -0,0 +1,27 @@
+id: good-env-config
+apis:
+- base_path: /v1
+  authentication:
+    jwt:
+      name: foo
+      issuer: bar
+      in:
+      - header: header
+      remote_jwks:
+        url: url
+        cache_duration: 1h
+  consumer_authorization:
+    in:
+    - header: x-api-key
+  operations:
+  - name: op-1
+    http_match:
+    - path_template: /petstore
+      method: GET
+  - name: op-2
+    http_match:
+    - path_template: /bookshop
+      method: POST
+  http_request_transforms:
+    set_headers:
+      x-apigee-target: target
```

Comment:
JFYI we'll probably use `x-apigee-route` ... not super important for test data, just wanted to share since it's relevant